app/models/sipity/workflow.rb

```diff
@@ -10,16 +10,40 @@ class Workflow < ActiveRecord::Base
     has_many :workflow_states, dependent: :destroy, class_name: 'Sipity::WorkflowState'
     has_many :workflow_actions, dependent: :destroy, class_name: 'Sipity::WorkflowAction'
     has_many :workflow_roles, dependent: :destroy, class_name: 'Sipity::WorkflowRole'
-    has_many :permission_templates, dependent: :restrict_with_exception, class_name: 'Hyrax::PermissionTemplate'
+    belongs_to :permission_template, class_name: 'Hyrax::PermissionTemplate', required: true
```

Comment:
Add a comment here about this just being one of the potential workflows that could be selected for that permission_template.